src/engine/QueryExecutionTree.h

```diff
@@ -232,14 +227,18 @@ class QueryExecutionTree {
     return getRootOperation()->getPrimarySortKeyVariable();
   }
 
+  // _____________________________________________________________
+  friend void PrintTo(const QueryExecutionTree& tree, std::ostream* os) {
+    auto& s = *os;
+    s << nlohmann::ordered_json{tree.getRootOperation()->runtimeInfo()}.dump(2);
+  }
+
  private:
   QueryExecutionContext* _qec;  // No ownership
   std::shared_ptr<Operation>
       _rootOperation;  // Owned child. Will be deleted at deconstruction.
   OperationType _type;
-  string _asString;
-  size_t _indent = 0;  // the indent with which the _asString repr was formatted
-  size_t _sizeEstimate;
+  size_t _sizeEstimate = std::numeric_limits<size_t>::max();
```

Comment:
Is this necessary? It does look wrong in the runtime information, for example for this query: https://qlever.cs.uni-freiburg.de/wikidata/G0I3i2